manifests/admin.pp

```diff
@@ -40,6 +40,20 @@
 # @param default_admin_sudo_cmnds
 #   The set of commands that ``$admin_group`` should be able to run by default
 #
+# @param admin_sudo_options
+#   A hash of sudo options to give to all admin sudo commands
+#
+# @example
+#   simp::admin::admin_sudo_options:
+#     role: 'unconfined_r'
+#
+# @param auditor_sudo_options
+#   A hash of sudo options to give to all auditor sudo commands
+# 
+# @example
+#   simp::admin::auditor_sudo_options:
+#     role: 'staff_r'
```

Comment:
This *might* also need to be `unconfined_r` but I'm not sure.

If the original user doesn't have `staff_r`, they can't even run `su` or `sudo`.

Reply:
I changed the example since you're right, they'll most likely need it to do anything with selinux on. I did not, however, make it default since it is for auditors rather than administrators.